src/DocumentDB.ChangeFeedProcessor.IntegrationTests/V3CompatibilityTests.cs

```diff
@@ -198,6 +212,39 @@ public async Task Schema_OnV2MigrationMaintainLeaseToken()
             }
         }
 
+        private async Task WaitUntilLeaseStoreIsInitializedAsync()
```

Comment:
ideally, this would take a timeout value

Reply:
Used a CancellationTokenSource with a timeout